src/layout.js

```diff
@@ -0,0 +1,68 @@
+import _ from 'lodash';
+import React, {Component, PropTypes} from 'react';
+import shouldComponentUpdate from 'react-addons-shallow-compare';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import Row from 'muicss/lib/react/row';
+import Col from 'muicss/lib/react/col';
+import Panel from 'muicss/lib/react/panel';
+
+import ConfirmDialog from './components/confirm_dialog';
+import PhotoGallery from './components/photo_gallery';
+
+import {showDialog} from './actions/dialog';
+
+const generatePhotos = (amount = 20) => {
+  return _.map(_.range(0, amount), id => ({
+    id,
+    fileUrl: `https://unsplash.it/200/300/?image=${id}`
+  }));
+}
+
+class Layout extends Component {
+
+  static propTypes () {
+    return {
+      showDialog: PropTypes.func
+    };
+  }
+
+  constructor () {
+    super();
+    this.onProfileGalleryUpdate = this.onProfileGalleryUpdate.bind(this);
+    this.shouldComponentUpdate = shouldComponentUpdate.bind(this);
+  }
+
+  onProfileGalleryUpdate (images) { // eslint-disable-line no-unused-vars
```

Comment:
I actually didn't know about `eslint-disable-line`, I've always used `eslint-disable` and `eslint-enable`. Awesome!